jwst/regtest/test_associations_sdp_pools.py

```diff
@@ -34,7 +34,7 @@
 SPECIAL_POOLS = {
     'jw00016_20230331t130733_pool': {
```

Comment:
There appears to be nothing special anymore about this pool. Suggest removing it completely from the dict.

Reply:
OK, I've removed it in the latest commit.